model_analyzer/perf_analyzer/perf_config.py

```diff
@@ -340,19 +342,22 @@ def to_cli_string(self):
 
         # single dashed options, then verbose flags, then main args
         args = []
-        args.extend(self._parse_short_options())
+        args.extend(self._parse_short_options(exclude_model_name))
         args.extend(self._parse_verbose_options())
         args.extend(self._parse_long_options())
 
         return " ".join(args)
 
-    def _parse_short_options(self):
+    def _parse_short_options(self, exclude_model_name: bool = False) -> List:
         """
         Parse the perf analyzer single dash options
         """
         temp_args = []
         for key, value in self._options.items():
             if value:
+                if exclude_model_name and key == "-m":
```

Comment:
I see you checking short options. Could model-name be in the long options?

Reply:
No, model name (`-m`) is something MA adds in when it creates the PA config.